service-mesh-probe/service-mesh.proto

```diff
@@ -56,6 +56,9 @@ message ServiceMeshMetric {
     // The sidecar/proxy internal error code, the value bases on the implementation.
     // The envoy internal error codes are listed here, https://www.envoyproxy.io/docs/envoy/latest/api-v2/data/accesslog/v2/accesslog.proto#data-accesslog-v2-responseflags
     string internalErrorCode = 14;
+
+    // The throughput detail of the mesh metrics.
+    Throughput throughput = 15;
```

Comment:
What is the relationship between TCP and HTTP about the traffic? Does TCP have latency?

Reply:
My point is, TCP should be separate object or only an extra field.

Reply:
Nearly all the current fields (except for `endpoint`, `responseCode`, `status`) exist in TCP, I can't come out with a better place to put `throughput` atm.

Reply:
But we have OAL related to the latency, if you share the existing one, I think the metrics there would be wrong.

What do you think? Should we have ServiceMeshTCPMetric?

Reply:
> But we have OAL related to the latency, if you share the existing one, I think the metrics there would be wrong.

Both of Tcp and Http share the same field `latency`, that said, for TCP, `latency = round trip time`, for HTTP it's what it is now, do you mean the latency would be counted twice for Tcp services? If so, it is not the case.

Reply:
> do you mean the latency would be counted twice for Tcp services? If so, it is not the case.

Yes. that was my question. So, it is fine.

Reply:
Let's rename this to TCP?

Reply:
> Let's rename this to TCP?

Do you mean `tcpThroughput`?

Reply:
I don't want throughput or traffic. As this is an access metric, it is basically duplicated information. 
Also, if one day, we need to add more fields related to TCP, it is better naming as TCP directly, such as resent packages/bytes(guessing only)

Reply:
OK hope I understand correctly, do you mean rename `Throughput` to `TCP`, and make `throughput` a field of `TCP`?

```
message ServiceMeshMetric {
  //...
  TCP tcp = 15;
  //...
}

message TCP {
  Throughput throughput = 1;
}
```

Reply:
Whether do we need `Throughput`, seems not very important.

Reply:
Like this?

```
message ServiceMeshMetric {
  //...
  TCP tcp = 15;
  //...
}

message TCP {
  // The received data in bytes
  int64 received = 1;
  // The sent data in bytes
  int64 sent = 2;
}
```

Reply:
Yes, is this good to you?

Reply:
> Yes, is this good to you?

Good to me